From 226f590f1f6907c7672ecbb901047ad9edbd91c1 Mon Sep 17 00:00:00 2001
From: WANG Xuerui <git@xen0n.name>
Date: Tue, 13 May 2025 14:07:27 +0800
Subject: [PATCH 2/2] Revert "Fix, need to prevent Anaconda and poetry without
 virtualenv being used in a mixed fashion"

This reverts commit 00f649851d9464f7baa07dea7ce753a0635ad3b9.
---
 nuitka/utils/Distributions.py | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/nuitka/utils/Distributions.py b/nuitka/utils/Distributions.py
index 63bef253b..f309e94b5 100644
--- a/nuitka/utils/Distributions.py
+++ b/nuitka/utils/Distributions.py
@@ -553,14 +553,6 @@ def getDistributionInstallerName(distribution_name):
 
             if installer_name:
                 _distribution_to_installer[distribution_name] = installer_name
-
-                if installer_name.lower().startswith("poetry"):
-                    metadata_logger.sysexit(
-                        """\
-Error, cannot use poetry and conda combined in a virtualenv, due \
-to poetry corrupting installer information. Use either pure conda \
-or poetry virtualenv."""
-                    )
             elif isAnacondaPython():
                 _distribution_to_installer[distribution_name] = "conda"
             elif isPdmPackageInstallation(distribution):
-- 
2.48.1

